specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2018-10-01/imds.json

```diff
@@ -0,0 +1,731 @@
+                    "type":"string",
+                    "description":"This is the MAC address of the interface."
+                }
+            }
+        },
+        "planProperties": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "description": "This is the Plan ID."
+                },
+                "publisher": {
+                    "type": "string",
+                    "description": "This is the publisher ID."
+                },
+                "product": {
+                    "type": "string",
+                    "description": "This is the product of the image from the Marketplace."
+                }
+            }
+        },
+        "publicKeysProperties": {
+            "type": "object",
+            "properties": {
+                "path": {
+                    "type": "string",
+                    "description": "This specifies the full path on the VM where the SSH public key is stored."
+                },
+                "keyData": {
+                    "type": "string",
+                    "description": "This is the SSH public key certificate used to authenticate with the VM."
+                }
+            }
+        },
+        "ipv4Properties": {
+            "type": "object",
+            "properties": { 
+                "privateIpAddress": {
+                    "type":"string",
+                    "description":"This is the private IP address assigned to the interface."
+                 },
+                "publicIpAddress": { 
+                    "type":"string",
+                    "description":"This is the public IP address assigned to the interface."
+                }
+            }
+        },
+        "ipv6Properties": {
+            "type": "object",
+            "properties": { 
+                "privateIpAddress": {
+                    "type":"string",
+                    "description":"This is the private IPV6 address assigned to the interface."
+                }
+            }
+        },
+        "subnetProperties": {
+            "type": "object",
+            "properties": { 
+                "address": {
+                    "type":"string",
+                    "description":"This is the address range of the subnet."
+                },
+                "prefix": { 
+                    "type":"string",
+                    "description":"This is the prefix of the subnet."
+                }
+            }
+        },
+        "instance": {
+            "type": "object",
+            "properties": {
+                "compute": {
+                    "$ref": "#/definitions/compute",
+                    "description": "Compute Metadata"
+                },
+                "network": {
+                    "$ref": "#/definitions/network",
+                    "description": "Network Metadata" 
+                }
+            }
+        },
+        "attested": {
```

Comment:
PascalCase name and provide meaningful description

Reply:
Changed to Attested and added description